Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -417,28 +422,31 @@ public static Furniture PlaceInstance(Furniture proto, Tile tile)
     /// <param name="deltaTime">The time since the last update was called.</param>
     public void Update(float deltaTime)
     {
-        if (PowerConnection != null && PowerConnection.IsPowerConsumer && HasPower() == false)
+        if (isBeingDestroyed == false)
```

Comment:
This isn't preventing new jobs to be created. Most new jobs are created from Lua, when OnUpdate is trigger from EventAction. And a furniture should still stop working when it has no power and is set to be deconstructed.

If we want to prevent new jobs to be added, we can do so in FurnitureJobs.Add. I think that a furniture could still work until it gets deconstructed, but maybe it shouldn't be used. For beds, we could not use the ones that will be deconstructed.